### ISSUE TITLE
Documentation outdated

### ISSUE THREAD

**Issue by rojalator** [openvax/mhcflurry#236]
The documentation referred to (http://openvax.github.io/mhcflurry/) is out-of-date - it is for 2.0.0 whereas the current version is 2.1.1 (as installed by pip).

As an example, the command-line reference for `mhcflurry-predict-scan` shows `--results-filtered` as an option but running the command results in:

`mhcflurry-predict-scan: error: unrecognized arguments: --results-filtered affinity`

The entry in the command-line tutorial is also incorrect due to this.

This option was removed in https://github.com/openvax/mhcflurry/commit/20918307b07fc337b7eec85ad88c12f8c45ba289 following https://github.com/openvax/mhcflurry/issues/219

**Comment by iskandr** [comment]
@timodonnell ?

**Comment by timodonnell** [comment]
Yeah, things have drifted since the docs were written. Updating them is a to do item.
